Codecademy/discourse-codebytes-plugin
# add prettier+eslint rules with commit hook

## Overview
gotta look professional

### PR Checklist

### Notes
mostly copied configs over from static-sites, with a few changes that make more sense here (mostly changing which paths we're linting and ignoring ts/css/scss/etc).

### Testing Instructions
1. run discourse locally with this code
2. make sure it don't asplode
3. make a change that definitely has bad formatting, commit it, and see that prettifier fixes it on commit
